src/WebJobs.Script/OutOfProc/Rpc/RpcInitializationService.cs

```diff
@@ -164,6 +175,14 @@ internal bool ShouldStartInPlaceholderMode()
             return false;
         }
 
+        internal bool ShouldStartAsPlaceholderPool()
+        {
+            // We are in placeholder mode but a worker runtime IS set
+            return _environment.IsPlaceholderModeEnabled()
+                && !string.IsNullOrEmpty(_workerRuntime)
+                && _placeholderPoolWhitelistedRuntimes.Contains(_workerRuntime);
```

Comment:
>_placeholderPoolWhitelistedRuntimes.Contains(_workerRuntime); [](start = 19, length = 61)

Do we need to worry about casing here?

Reply:
apologies. this looks like a merge commit. But good catch we should fix this in dev branch. Will send a PR. FYI @mhoeger